drivers/sensor/am2301b/am2301b.c

```diff
@@ -0,0 +1,229 @@
+/* am2301b.c - Driver for Aosong AM2301B Humidity and Temperature Module */
+/*
+ * Copyright (c) 2022 Hisanori Yamaoku
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT aosong_am2301b
+
+#include <device.h>
+#include <sys/util.h>
+#include <sys/byteorder.h>
+#include <zephyr/types.h>
+#include <devicetree.h>
+#include <drivers/i2c.h>
+#include <drivers/sensor.h>
+#include <logging/log.h>
```

Comment:
Please add zephyr/ to the include paths. i.e., `#include <zephyr/logging/log.h>`

Reply:
fixed it